bibliosearch_backend/bibliosearch_backend/urls.py

```diff
@@ -20,9 +20,9 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('searchBook', views.search_book)
```

Comment:
what about `book/search/` for the endpoint

this way when we have other endpoints for book it will be consistent